I would suggest to make _createLoremIpsumObject public accessible

failnix opened the pull request:
otherwise all custom inherited objectFactories have to be placed under the package org.bbottema.loremipsumobjects.typefactories

failnix committed:
I would suggest to make _createLoremIpsumObject public accessible
otherwise all custom inherited objectFactories have to be placed under
the package org.bbottema.loremipsumobjects.typefactories

bbottema commented:
Thank you for contributing! 

The method is for internal use only, so then `protected` seems to be the appropriate scope, rather than opening it up completely.  If you update it, I'll merge the PR!

failnix committed:
Refactoring LoremIpsumObjectFactory, RandomBigDecimalFactory

* LoremIpsumObjectFactory#_createLoremIpsumObject with
protected-visibility
* added Config-Option for fixed-scaling in RandomBigDecimalFactory

failnix commented:
> Thank you for contributing!
> 
> The method is for internal use only, so then `protected` seems to be the appropriate scope, rather than opening it up completely. If you update it, I'll merge the PR!

Hi I changed the method-signature from public to protected and added a possibility to config scale-size of BigDecimalFactory .

bbottema commented:
Change released in 4.2.0. Enjoy!
